C#/cs01/14.cs

```diff
@@ -0,0 +1,42 @@
+using System;
```

Comment:
why do we have strategy program and stack program combined in single PR

Reply:
by mistake sorry